src/test/java/com/epam/tc/tests/TrelloFunctionTests.java

```diff
@@ -0,0 +1,59 @@
+package com.epam.tc.tests;
+
+import com.epam.tc.ConfProperties;
+import com.epam.tc.TestBase;
+import com.epam.tc.dto.BoardDto;
+import java.util.ArrayList;
+import java.util.List;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class TrelloFunctionTests extends TestBase {
+
+    public static List<BoardDto> boardDtosListToDelete = new ArrayList<>();
```

Comment:
It's better to move this list to another class, e.g. create a class for storing all data that should be deleted (it can be useful if not only these boards or other DTOs should be removed)